src/tests/gssapi/t_gssapi.py

```diff
@@ -143,22 +143,23 @@
 realm.run(['./t_inq_cred', '-k', '-b'], expected_msg=realm.host_princ)
 
 # Test gss_export_name behavior.
-out = realm.run(['./t_export_name', 'u:x'])
-if out != '0401000B06092A864886F7120102020000000D78404B5242544553542E434F4D\n':
+output = realm.run(['./t_export_name', 'u:x'])
```

Comment:
Yes, but: we lose the explanation of what the tests are.  If we're fine with that I'll make the change.